docs/reference/troubleshooting/common-issues/red-yellow-cluster-status.asciidoc

```diff
@@ -74,35 +74,31 @@ A shard can become unassigned for several reasons. The following tips outline th
 most common causes and their solutions.
 
 [discrete]
-[[fix-cluster-status-reenable-allocation]]
-===== Re-enable shard allocation
+[[fix-cluster-status-only-one-node]]
```

Comment:
careful with changing anchors without moving them to another spot. this can break links. we'll see if the docs build without errors